notebook.ipynb

```diff
@@ -0,0 +1,174 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Clox"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "----------------"
+   ]
+  },
```

Comment:
I don't like this block. Either remove it or merge it with the previous one. It feels like there were something there which we removed and forgot to clean the rest.

Reply:
Good point

Reply:
fixed in https://github.com/sepandhaghighi/clox/pull/73/commits/68bcb56e8cff9000a82fe10d1eee48740df96e86